src/styles/_bootstrap.scss

```diff
@@ -0,0 +1,8239 @@
+/*!
```

Comment:
@akasrai bootstrap has to be renamed to sass as sass-compiler taking the absolute path of machine